chapters/560_environment-and-setup.md

```diff
@@ -44,33 +43,33 @@ another challenge. The subsequent approach is a first and more general solution
 modern mobile device and on this device a *PDaaS* mobile application is installed.*
 
 1.  After starting the app, the operator needs to establish a connection between server and mobile 
-    application. Therefor the operator either has to scan a QR-Code with the help of that app. The 
+    application. Therefore, the operator has to scan a QR-Code with the help of that app. The 
     QR-Code is presented to the operator within the management tool of the *PDaaS* running in a 
-    browser. Or the operator inserts her credentials into a form presented by the mobile 
+    browser. Alterntively, the operator inserts her credentials into a form presented by the mobile 
     application.
 
-2.  After the connection has established, the operator can trigger a progress that duplicated
+2.  After the connection is established, the operator can trigger a progress that duplicates
     all her personal data to the device that has just been associated with the *PDaaS*.
     
-3.  At this point, one of two ways can be proceeded with, depending on whether a complete write log
-    for the *personal data* ([see discussion about backup strategies](#data) does 
-    exist or not.
+3.  At this point, one of two ways path can follow, depending on whether a complete write log
```

Comment:
regarding "one of two ways path can follow,", isnt that doubled somehow? or should it read "one of two paths can follow,"?